wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/devfile/DevfileService.java

```diff
@@ -166,9 +153,7 @@ public Response createFromWorkspace(
           String key)
       throws NotFoundException, ServerException, BadRequestException, ConflictException,
           JsonProcessingException {
-    validateKey(key);
-    return Response.ok()
-        .entity(objectMapper.writeValueAsString(devfileManager.exportWorkspace(key)))
-        .build();
+    throw new BadRequestException(
+        "This is an outdated method please use /api/workspace/devfile to create a new workspace");
```

Comment:
Run-on sentence; does not explain WHY user should use other api.

Preferred: "This method is deprecated and may be removed at any time. Please use ..."